src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html

```diff
@@ -3,3 +3,20 @@
         ${contribSelectFragmentsHtml}
     </select>
 </div>
+<div>
+    <a href='#' data-toggle="modal" data-target="#more-info-equal-share-modal">${equalShareHelp}</a>
```

Comment:
Use `"` instead of `'`.
Also, `href="#"` is not the correct way to suppress an `<a>` tag because it will take the browser screen to the top of the web page (unless it is the intention). The correct way to do the former is `href="javascript:;"`.